src/Microsoft.TestPlatform.Extensions.BlameDataCollector/WindowsHangDumper.cs

```diff
@@ -114,64 +116,50 @@ internal static void CollectDump(Process process, string outputFile, DumpTypeOpt
 
         EqtTrace.Verbose($"WindowsHangDumper.CollectDump: Selected dump type {type}. Dumping {process.Id} - {process.ProcessName} in {outputFile}. ");
 
-        if (RuntimeInformation.OSArchitecture == System.Runtime.InteropServices.Architecture.X86)
```

Comment:
It also made the rest of the code complicated. Now we just check if the current process is the same, and if not we start minidump tool, because the pInvoke call needs to happen from a process with the correct bitness.